### ISSUE TITLE
Pinpoint: createCampaign returns too many active campaigns but all campaigns are deleted.

### ISSUE THREAD

**Issue by njdullea** [aws/aws-sdk-js#3183]
- [X] I've gone through [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/welcome.html) and [API reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/)
- [X] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js) for answers
- [X] I've searched for [previous similar issues](https://github.com/aws/aws-sdk-js/issues) and didn't find any solution
  
**Describe the bug**
I am trying to programmatically setup and takedown campaigns. I have created lots of campaigns over the last couple of days but deleted them all. When I try to create new campaigns now I receive an error: 'Too many active campaigns'; however, all the campaigns I have created have a status of 'deleted' so they should not count as active campaigns.

**Is the issue in the browser/Node.js?**
Node.js

**If on Node.js, are you running this on AWS Lambda?**
No

**Details of Node.js version**
Node version: 12.16.0

**SDK version number**
v2.645.0

**To Reproduce (observed behavior)**
Here is a cleaned [repo](https://github.com/njdullea/pinpointInteractionScripts) as an example of my setup. Instructions are in the readme to change and use it yourself.
- Create 200 or more campaigns and delete them all. 
- Try to create more and it returns too many active campaigns.

**Expected behavior**
The  [documentation](https://docs.aws.amazon.com/pinpoint/latest/developerguide/quotas.html) says that a campaign is active if it has a status of 'SCHEDULED', 'EXECUTING' or 'PENDING_NEXT_RUN'. All of my campaigns have a status of 'DELETED' so they should not be counted as active, and there should not be a warning saying I have too many active campaigns.

**Screenshots**
Error saying too many active campaigns:
<img width="291" alt="Screen Shot 2020-03-26 at 10 48 49 AM" src="https://user-images.githubusercontent.com/29107642/77666857-7983c600-6f4f-11ea-945b-e87263f8b2df.png">

List of campaigns ids and their status (not all of the campaigns, but should be enough)
<img width="469" alt="Screen Shot 2020-03-26 at 10 49 10 AM" src="https://user-images.githubusercontent.com/29107642/77666940-98825800-6f4f-11ea-9f83-6af7896adadf.png">

**Comment by HenrikJohnson** [comment]
I have not been able to reproduce this behavior. That said if the campaigns you created were actually executing at the time of deletion the quota for creating new campaigns would not be available until the campaign executions had actually completed (Even if the status is DELETED the campaigns might still be running in the back end).

The campaigns should become available relatively shortly after they were DELETED and as they complete.

**Comment by njdullea** [comment]
I haven't made or removed any campaigns since I submitted this bug report.

I went and tried to manually make a campaign in the aws console and still received this error: 
<img width="1207" alt="Screen Shot 2020-03-27 at 5 47 50 PM" src="https://user-images.githubusercontent.com/29107642/77806630-39a60700-7053-11ea-9377-2212a3e5335d.png">

To show I have no campaigns: 
<img width="975" alt="Screen Shot 2020-03-27 at 5 48 07 PM" src="https://user-images.githubusercontent.com/29107642/77806656-4cb8d700-7053-11ea-8d2c-1a257b36ebf0.png">

Presumably, since the error came up in the console as well this is not an aws-sdk-js issue, but rather a general pinpoint issue.

In any case, I'm moving away from campaigns/segments and just gonna use the pinpoint direct send message with my own targeting implementation. I thought y'all might still wanna check it out in case it is an actual bug, but if you are not able to recreate then it should be all good!

Thanks for looking into it @HenrikJohnson !
